.goreleaser.yml

```diff
@@ -4,6 +4,8 @@ builds:
   goos:
   - linux
   - darwin
+  ldflags:
+  - -X=github.com/windmilleng/tilt/internal/cli.version={{ .Version }}
```

Comment:
you're aware that goreleaser already does this automatically, right?

it sets this variable
https://github.com/windmilleng/tilt/blob/master/cmd/tilt/main.go#L11

the whole point of the internal/cli variable you're setting here is because we see lots of people build from source with `go get -u`.

This change defeats the whole point of that variable, because now it's just equivalent to the other variable.